openslides_backend/services/datastore/interface.py

```diff
@@ -12,7 +12,7 @@
 PartialModel = Dict[str, Any]
 Found = TypedDict("Found", {"exists": bool})
 Count = TypedDict("Count", {"count": int})
-Aggregate = Dict[str, Any]  # TODO: This interface seams to be wrong.
+OptionalInt = Optional[int]
```

Comment:
This is just confusing ^^ please just write the full type where you use it.

Reply:
Done